WordPress/gutenberg
# Template Part Placeholder - Try more descriptive and contextual button labels.

## Description

Related to discussion in https://github.com/WordPress/gutenberg/issues/29950.  The template part placeholder's button names are not very descriptive for a new user to FSE.  "Choose existing" for example could be easily confused with wanting to choose an existing block pattern for their template part.

Here, we look at changing the copy "Choose existing" to "Connect this block to an existing %s" (%s as 'header', 'footer', etc.).  And update "Create %s" to "Create a new %s" with the added "(patterns available)" when applicable.

![Screen Shot 2021-07-30 at 1 01 50 PM](https://user-images.githubusercontent.com/28742426/127687432-e5d09888-2576-4bc8-acb8-dd13c9363961.png)

![Screen Shot 2021-07-30 at 1 01 55 PM](https://user-images.githubusercontent.com/28742426/127687436-ab1bd02b-0d81-4ae8-8132-f6def1b63782.png)


## How has this been tested?



* test the template part placeholders in both editors.
* Verify the copies work as expected.
* For testing, add a template part specific pattern (you can copy a pattern in `lib/block-patterns`, give it a new name, and change its blockTypes to 'core/template-part/header'.).
* Verify the extra " (patterns available)" text appears when applicable.

## Screenshots 

## Types of changes





## Checklist:
